### ISSUE TITLE
[Failing Test] kind CI jobs are failing

### ISSUE THREAD

**Issue by alejandrox1** [kubernetes/kubernetes#82097]
**Which jobs are failing**:
kind CI jobs are failing.
For example:
* ci-kubernetes-kind-e2e-parallel
* pull-kubernetes-conformance-kind-ipv6
* ci-kubernetes-kind-ipv6-e2e-parallel

**Since when has it been failing**:
Jobs started failing after https://github.com/kubernetes/kubernetes/pull/81903

**Testgrid link**:
* https://testgrid.k8s.io/sig-release-master-blocking#kind-master-parallel
* https://testgrid.k8s.io/sig-release-master-blocking#kind-ipv6-master-parallel
* https://testgrid.k8s.io/conformance-all#conformance-image,%20master%20%28dev%29

**Reason for failure**:

Job failing on
```
kind build node-image --type=bazel --kube-root=/go/src/k8s.io/kubernetes
```
see https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-conformance-image-test/1166803875143880704#0:build-log.txt%3A553

The error message is as follows:
```
0828 20:10:25.963] Pulling: W0828 20:10:25.905084      79 defaults.go:196] The recommended value for "rotate-certificates" in "" is: true; the provided value is: false
I0828 20:10:25.963] Pulling: k8s.gcr.io/etcd:3.3.10
I0828 20:10:25.963] Pulling: k8s.gcr.io/coredns:1.5.0
I0828 20:10:25.964] Pulling: k8s.gcr.io/pause:3.1
I0828 20:10:29.211] time="20:10:29" level=warning msg="Failed to pull W0828 20:10:25.905084      79 defaults.go:196] The recommended value for \"rotate-certificates\" in \"\" is: true; the provided value is: false with error: exit status 1"
I0828 20:10:39.992] time="20:10:39" level=error msg="Image build Failed! Failed to pull Images: exit status 1"
W0828 20:10:41.774] Error: error building node image: exit status 1
```

The failure on the kubeadm side seems to be related to https://github.com/kubernetes/kubernetes/pull/81903/files#diff-5d5570f188cf5eb502aa81ba486fc736R178

There is already a proposed fix on the kind side, see https://github.com/kubernetes-sigs/kind/pull/825

/milestone v1.16
/priority critical-urgent
/kind failing-test
/sig testing
/cc @BenTheElder @neolit123

**Comment by alejandrox1** [comment]
kubeadm fix https://github.com/kubernetes/kubernetes/pull/82103

**Comment by alejandrox1** [comment]
reopen to track kubeadm fix #82103
/reopen

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/kubernetes/pull/82103 landed
